### PR TITLE
[MIG][14.0] account_banking_mandate

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

File: account_banking_mandate/migrations/14.0.1.0.0/post-migrate.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Opener B.V. - Stefan Rijnhart <stefan@opener.amsterdam>
```
Comment (CarlosRoca13) [inline account_banking_mandate/migrations/14.0.1.0.0/post-migrate.py]: Please update the year, I don't want to remember de 2020 :laughing:
```suggestion
# Copyright 2021 Opener B.V. - Stefan Rijnhart <stefan@opener.amsterdam>
```

**Review by CarlosRoca13 — APPROVED** [review]
LGTM just a minor comment

File: account_banking_mandate/README.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+This module depends on :
+* account_payment
+
+This module is part of the OCA/bank-payment suite.
+
+Usage
+=====
+
+To use this module, see menu "Invoicing/Accounting > Payments > Debit Orders"
```
Comment (CarlosRoca13) [inline account_banking_mandate/README.rst]:
```suggestion
To use this module, see menu "Invoicing/Accounting > Customers > Debit Orders"
```

**Review by joao-p-marques — APPROVED** [review]
Thank you @StefanRijnhart Looks good :+1: 
About the commit history, could you squash the 2 migration commits in one with the appropriate format? (`[MIG] $MODULE: Migration to 14.0` - See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0) And possibly put the `[MIG]` commit after the `[IMP]` one?

**Comment by joao-p-marques** [comment]
@StefanRijnhart do you still plan on making more changes? If not, can you mark the PR as ready?

**Comment by StefanRijnhart** [comment]
I can follow up, but I don't agree with squashing the 2 commits because one is from Alexis and the other is mine.

**Comment by joao-p-marques** [comment]
> I can follow up, but I don't agree with squashing the 2 commits because one is from Alexis and the other is mine

Ok, but you can make a co-authored commit between both: https://github.blog/2018-01-29-commit-together-with-co-authors/

Also, could you add the `[IMP]` prefix in the `black, isort, etc...` commit?

This is nitpicking, however, not that important. :wink:

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
When commits come from several authors, it's not needed to squash them. The squashing activity is for useless administrative commits or commits from same author originated by iterative migration.

Merging:

/ocabot merge nobump

**Comment by joao-p-marques** [comment]
Ok, thank you for the explanation 👍

**Comment by StefanRijnhart** [comment]
Oh, I just pushed the coauthored commit but now this one is merged already. Code was identical so it's not a problem. Thanks for the pointer though, @joao-p-marques

**Pull request merged**